api/AudioContext.json

```diff
@@ -89,20 +89,34 @@
           "mdn_url": "https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext",
           "description": "<code>AudioContext()</code> constructor",
           "support": {
-            "chrome": {
-              "version_added": "55",
-              "notes": [
-                "Each tab is limited to 6 audio contexts in Chrome; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Chrome'>Per-tab audio context limitation in Chrome</a>.",
-                "If <code>latencyHint</code> isn't valid, Chrome throws a <code>TypeError</code> exception. See <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Non-standard_exceptions_in_Chrome'>Non-standard exceptions in Chrome</a> for details."
-              ]
-            },
-            "chrome_android": {
-              "version_added": "55",
-              "notes": [
-                "Each tab is limited to 6 audio contexts in Chrome; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Chrome'>Per-tab audio context limitation in Chrome</a>.",
-                "If <code>latencyHint</code> isn't valid, Chrome throws a <code>TypeError</code> exception. See <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Non-standard_exceptions_in_Chrome'>Non-standard exceptions in Chrome</a> for details."
-              ]
-            },
+            "chrome": [
+              {
+                "version_added": "35",
+                "notes": [
+                  "Prior to Chrome 66, each tab is limited to 6 audio contexts in Chrome; attempting to create more will throw a <code>DOMException</code>. For details see <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Per-tab_audio_context_limitation_in_Chrome'>Per-tab audio context limitation in Chrome</a>.",
+                  "If <code>latencyHint</code> isn't valid, Chrome throws a <code>TypeError</code> exception. See <a href='https://developer.mozilla.org/docs/Web/API/AudioContext/AudioContext#Non-standard_exceptions_in_Chrome'>Non-standard exceptions in Chrome</a> for details."
+                ]
+              },
+              {
+                "version_added": "14",
+                "version_removed": "57",
```

Comment:
Removal of `webkitAudioContext` in Chromium 57 is supported by Web Confluence data:
http://web-confluence.appspot.com/#!/catalog?releases=%5B%22Chrome_55.0.2883.75_OSX_10.11.6%22,%22Chrome_55.0.2883.75_Windows_10.0%22,%22Chrome_55.0.2883.84_Android_6.0.1%22,%22Chrome_56.0.2924.76_OSX_10.11.6%22,%22Chrome_56.0.2924.87_Android_6.0.1%22,%22Chrome_56.0.2924.87_Windows_10.0%22,%22Chrome_57.0.2987.108_Android_6.0.1%22,%22Chrome_57.0.2987.110_Windows_10.0%22,%22Chrome_57.0.2987.98_OSX_10.11.6%22,%22Chrome_58.0.3029.81_OSX_10.11.6%22,%22Chrome_58.0.3029.81_Windows_10.0%22,%22Chrome_58.0.3029.83_Android_6.0.1%22,%22Chrome_85.0.4183.83_Windows_10.0%22%5D&q=%22webkitaudiocontext%22